src/exporters/MMDExporter.ts

```diff
@@ -167,13 +123,53 @@ const MMDExporter = function () {
 
     return outputShiftJis === true ? unicodeToShiftjis(lines) : lines
   }
+}
 
-  /* TODO: implement
-	// animation + skeleton -> vmd
-	this.parseVmd = function ( object ) {
+// Unicode to Shift_JIS table
+let u2sTable: { [key: string]: number | undefined } | undefined
 
-	};
-	*/
+function unicodeToShiftjis(str: string): Uint8Array {
```

Comment:
If these are used internally by the class and not exported my opinion is they should be private methods. What do you think? The same goes for the other function `getBindBones`

Reply:
It is good to make all functions used in the class private methods.

Also, I would like to change `u2sTable` used by `unicodeToShiftjis` to a private property of the class.